### PR TITLE
Add function get inputs.conf

### PR DESCRIPTION
**Pull request by yenienserrano**
**Description:**
The agent status graph shows intervals where there are more active agents than registered agents.

**Resolve:**

A function is added to retrieve the data from the inputs.conf file to know the time of the interval and put the same time to the query used by the graph

**Steps to reproduce:**

1. Navigate to the agent page
2. See that the graph coincides with the number of agents

**Close:**

The agent status graph shows intervals where there are more active agents than registered agents [#1209](https://github.com/wazuh/wazuh-splunk/issues/1209)

**Screenshot:**

image

### CODE REVIEW

**Comment by AlexRuiz7** [comment]
Testing with 1 and then 2 agents
------------------------------

image

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark: 

Peek 2022-02-07 17-06

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js [hunk 1]
```diff
@@ -248,30 +248,33 @@ define([
 
       this.scope.getAgentStatus = () => {
         try {
-          this.clusOrMng = Object.keys(
-            this.currentDataService.getFilters()[0]
-          )[0]
+          const filters = this.currentDataService.getFilters()[0]
+          const nodeType = Object.keys(filters)[0]
+          var agentsStatusFilter = false
```
[the remark this answers is not in the document]
Comment (AlexRuiz7) [reply SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js]: Applied

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js [hunk 2]
```diff
@@ -248,30 +248,33 @@ define([
 
       this.scope.getAgentStatus = () => {
         try {
-          this.clusOrMng = Object.keys(
-            this.currentDataService.getFilters()[0]
-          )[0]
+          const filters = this.currentDataService.getFilters()[0]
```
[the remark this answers is not in the document]
Comment (AlexRuiz7) [reply SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agentsCtrl.js]: Applied

**Pull request merged**